src/libs/ontologyService.js

```diff
@@ -13,4 +13,16 @@ export async function searchOntology(obolibraryURL) {
   }
 }
 
+export function extractDOIDFromUrl(urls) {
+  const doidArr = [];
+  urls.forEach(url => {
+    const startIdx = url.search(/DOID_\d+/);
+    if (startIdx > -1) {
+      doidArr.push(url.slice(startIdx));
+    }
+  });
+
+  return urls;
```

Comment:
I was expecting to see doidArr returned here instead of urls.
Isn't this just passing the input through?

Reply:
No you're right, it needs to be doidArr.